docs/docs/features/keymaps.mdx

```diff
@@ -38,21 +37,18 @@ For the full set of possible behaviors, see the [overview page for behaviors](..
 ## Layers
 
 Like many mechanical keyboard firmwares, ZMK keymaps are composed of a collection of layers, with a
-minimum of at least one layer that is the default, usually near the bottom of the "stack". Each layer
+minimum of at least one layer that is the default, usually near the bottom of the "layer stack". Each layer
 in ZMK contains a set of bindings that bind a certain behavior to a certain key position in that layer.
 
 |                                                   ![Diagram of three layers](../assets/features/keymaps/layer-diagram.png)                                                    |
```

Comment:
This should really be replaced as well, at least with one that doesn't use the `KC_` prefix. But maybe for another PR.

Reply:
I am not artistically talented, so someone else will need to sort that out I think.

Reply:
Yeah, we could ask Thomas@splitkb to share the source (he was the original author) or reproduce it otherwise.